package.json

```diff
@@ -46,6 +46,7 @@
     "test:e2e": "npm run test:playwright -- test",
     "test:e2e-report": "npm run test:playwright -- show-report",
     "lint": "npx eslint . --report-unused-disable-directives",
+    "lint:prune": "! ts-prune | grep -v \"used in module\"",
```

Comment:
Once https://github.com/nadeesha/ts-prune/pull/202 lands, this can be simplified.

Reply:
don't mind the grep etc, tho we have some windows users. I'm more curious about `!` vs `npx`

Reply:
`!` inverts the exit code of the command it prepends. It's here because `grep -v` will exit with an error code if none are found. But we want none to be found!

On my machines, even without `ts-prune` installed globally, this runs without `npx` because it's an `npm run context`. But you can switch it to `! npx ts-prune ...` if I'm being silly & misconfiguring it!

Reply:
Well, if we could get away with just `ts-prune` then the `npx` wrapper would improve the life of windows users, however, since this is meant to only run in linux or CI environments I think it's fine with or without.

I'll resolve this conversation and think about how to document the scripts, once we figure out CI usage it's a check we will definitely want in some of our other typescript based repos! 🍕 💯 🚀

Reply:
> the `npx` wrapper would improve the life of windows users

Oh interesting, how would this do that? Do windows users not get the global script execution I've been enjoying?

Reply:
This is not going to sound great but not all of our new contributors read the docs, sometimes they clone the project and forget to `npm ci` - wrapping some binaries with `npx` gives a secondary execution prompt, enabling the user to see that something is not installed, then they install the binary for the script, most of the time that's a linter or our support tooling, telling them things like maybe their node version is not up to date and mentally hinting at the install process when they have unconsciously skipped that. 

Functionally what we are trying to do is have good tooling with the lowest possible entry bar for new contributors, technical or not, sometimes for example we end up wrapping even something like `ts-prune` through a static script, just so that we don't scope creep maintenance type tooling with bash skills requirement 😅 🍕

Reply:
The plan/dream is that open-sauced will provide a CLI that will magically help with all of those first blockers to contributing :D

Reply:
Oh thanks for the context, that's really helpful. I'll add it in!